### PR TITLE
add /searchCategory endpoint

### PR DESCRIPTION
**Pull request by notdb**
# Description

Adds a POST /searchCategory endpoint to search items by category

example:
{
	"category": "Cameras"
}

returns a list of all items with a category of cameras
## Checklist

Remove any items which are not applicable.

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

**Comment by notdb** [comment]
[Add endpoints for searching items](https://trello.com/c/CDHZ5JWc/47-add-endpoints-for-searching-items)

File: routes/items/itemsModel.js [hunk 1]
```diff
@@ -2,25 +2,49 @@ const knex = require('knex');
...
+
+function getItemByCondition(condition) {
+  return db('items').where(condition);
+}
+
+function getItemByZipCode(zipcode) {
+  return db('items').where(zipcode);
+}
+
+function getItemByCity(city) {
```
Comment (erik-sandoval) [inline routes/items/itemsModel.js]: Noticed in this function if a city was put in as lowercase or uppercase it returns two different sets of results.

Comment (notdb) [reply routes/items/itemsModel.js]: Taking a look

Comment (notdb) [reply routes/items/itemsModel.js]: This should be fixed now @erik-sandoval. Can you verify so I can merge the PR?

**Pull request merged**